File: learntoprogram/PackageDemoA.java
/**
* Description: 包技术
* 不同包间的访问,非继承的类需要加public权限才能访问
* @author fnfly2005
*/
package mypack;
public class PackageDemoA
{
    public void show()
    {
        System.out.println("Hello PackageDemoA!!");
    }

    protected void run()
    {
        System.out.println("PackageDemoA run!!");
    }
}

File: learntoprogram/pypack/PackageDemoA.py
#!/usr/bin/python
#coding:utf-8
"""
Description: 
"""
class PackageDemoA(object):
    def show(self):
        print "Hello PackageDemoA!!"

    def run(self):
        print "PackageDemoA run!!"

if __name__ == '__main__':
    p = PackageDemoA()
    p.show()
    p.run()
